packages/reactivity/src/collectionHandlers.ts

```diff
@@ -167,15 +167,19 @@ function createInstrumentations(
         }
       : {
           add(this: SetTypes, value: unknown) {
-            if (!shallow && !isShallow(value) && !isReadonly(value)) {
-              value = toRaw(value)
-            }
             const target = toRaw(this)
             const proto = getProto(target)
-            const hadKey = proto.has.call(target, value)
+            // 先获取原始值
```

Comment:
this comment seems unnecessary

Reply:
Yes, you're right. It seems that the map doesn't have this issue; let's first make modifications to the set in this PR.